chart/templates/support-bundle.yaml

```diff
@@ -0,0 +1,217 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  labels:
+    troubleshoot.sh/kind: support-bundle
+  name: {{ .Release.Name }}-support-bundle
+  namespace: {{ .Release.Namespace | quote }}
+stringData:
+    # To use it: kubectl support-bundle --load-cluster-specs --selector=carto.com/kind=support-bundle
+  support-bundle-spec: |
+    apiVersion: troubleshoot.sh/v1beta2
+    kind: SupportBundle
+    metadata:
+      name: {{ .Release.Name }}-support-bundle
+    spec:
+      collectors:
+        - clusterInfo: {}
```

Comment:
That could be filtered by namespace or doesn't sound logical? :thinking:

Reply:
the `clusterInfo` collector can't be filtered by namespace, it only show the kubernetes version and a few more parameters